.circleci/config.yml

```diff
@@ -93,10 +93,7 @@ jobs:
           command: dockerize -wait tcp://localhost:5432 -timeout 1m
       - run:
           name: Create test DB
-          command: |
```

Comment:
As a comment, we could maybe simplify all this in the future using YAML anchors (that `&default` and `*` stuff:

https://learnxinyminutes.com/docs/yaml/

Reply:
I don't think that's possible, since that's an array :(